### ISSUE TITLE
No accuracy improvements

### ISSUE THREAD

**Issue by AlexBlackNn** [Neerajj9/Text-Detection-using-Yolo-Algorithm-in-keras-tensorflow#5]
Hello. Thank you for your model.  
I have faced with a problem. I don't have any accuracy improvements during training, the loss is also hovering at the some value, with the accuracy  and loss being in the vicinity of 0.058 and  8.75 respectively

I have tried to use several optimizers: 
1)  Adam(lr=0.0001, beta_1=0.9, beta_2=0.999, epsilon=1e-08, decay=0.0) 
2) keras.optimizers.RMSprop(lr = 0.0001) 
3) keras.optimizers.RMSprop(lr = 0.001) 

and I try to change a batch size : 
1) 4
2) 8
3) 16

Actually, no effect
Could you help me to tackle this problem?

**Comment by Neerajj9** [comment]
Are you using the exact same architecture and dataset ?

**Comment by AlexBlackNn** [comment]
Yeah, I have utilized exactly the same architecture and dataset. However, I used google colab, so versions of Keras (2.3.1) and Tensorflow  (2.2.0-rc3) are different from yours in readme file.  	
	Having understood that, I downgraded keras and tensorflow versions, but I faced with other issues:
	1) my loss became ‘NaN’ and I solved that thanks to  @goldengrisha’s advice (https://github.com/Neerajj9/Text-Detection-using-Yolo-Algorithm-in-keras-tensorflow/issues/3#issuecomment-594271867)
	2) I failed to apply GPU calculation in colab with tensorflow 1.9.0 (no idea why, m.b.  I made some mistakes), and duration of one epoch have increased as 20 times as it was, so I have no information of the result related to that move
